theme.scss

```diff
@@ -13,8 +13,8 @@ $code-bg: #f8f5f0;
 $font-family-monospace: JuliaMono, Menlo, "Roboto Mono", "Lucida Sans Typewriter", "Source Code Pro", monospace;
 
 pre {
```

Comment:
I would avoid messing with the CSS and theme. It quickly gets into a matter of very personal preference and I generally want to avoid bikeshedding.

Reply:
Oh this change ensures that the Unicode based benchmark histograms line up so that there are no gaps in the two rows of text . I don’t think this was related to personal preference

Reply:
ah cool, then I'm onboard with that and good catch. :smile: 

One tip: you can "review"/comment on your PR diff beyond just responding to over people's comments. This is useful for providing additional context that we don't necessarily need to track long term. We do this at work a fair amount and it really helps! :heart:

Reply:
oh that's great! I will do that in the future. I guess I should also write longer commit messages? but the github UI doesnt really show the long commit message unless you click on it